web/tests/hotkey.test.js

```diff
@@ -21,9 +21,9 @@ const $ = require("./lib/zjquery");
 // it calls any external module other than `ui.foo`, it'll crash.
 // Future work includes making sure it actually does call `ui.foo()`.
 
-// Since all the tests here are based on narrow starting with all_messages.
+// Since all the tests here are based on narrow starting with message_feed.
 // We set our default narrow to all messages here.
```

Comment:
```suggestion
// We set our default narrow to message feed here.
```
Hello,

I hope you are doing well. I noticed this and thought it could be something to improve.

Thank you

Reply:
Thanks for pointing it out, I'll make this change